### PR TITLE
Update samples for preview.4

### LINKED ISSUE #131: Update samples to preview.4

### LINKED ISSUE #132: Rename eShopLite sample to AspireShop
It's not based on [eShop](https://github.com/dotnet/eshop) and the name currently just causes confusion.

### LINKED ISSUE #133: [dapr]  update version to `1.13.0-rc03` for preview4
Related to: https://github.com/Azure/azure-dev/issues/3321

dapr sample must be updated to version `1.13.0-rc03` to mitigate an issue with dapr sdk (https://github.com/dapr/dotnet-sdk/issues/1235)

In the meantime, you can switch to 1.13.0-rc03 of the Dapr NuGet packages which have the fix until the official 1.13 release.

### PR DESCRIPTION
**Pull request by DamianEdwards**
Need to revert changes to `nuget.config` once preview.4 is live on nuget.org before merge.

Fixes #131
Fixes #132
Fixes #133

### CODE REVIEW

File: samples/VolumeMount/VolumeMount.BlazorWeb/Data/CustomRetryingExecutionStrategy.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+public class CustomRetryingExecutionStrategy(ExecutionStrategyDependencies dependencies) : SqlServerRetryingExecutionStrategy(dependencies)
+{
+    protected override bool ShouldRetryOn(Exception exception)
+    {
+        if (exception is SqlException sqlException)
+        {
+            foreach (SqlError error in sqlException.Errors)
+            {
+                if (error.Number == 4060)
```
Comment (eerhardt) [inline samples/VolumeMount/VolumeMount.BlazorWeb/Data/CustomRetryingExecutionStrategy.cs:15]: Can we link to the bug that was logged for this?

**Review by eerhardt — APPROVED** [review]
LGTM (sans the NuGet.config change which shouldn't be needed once we ship)

**Comment by IEvangelist** [comment]
I think this is the wrong version number, correct me if I'm wrong but I've been using `8.0.0-preview.4.24129.7`.

**Comment by DamianEdwards** [comment]
@IEvangelist yeah it will need to be updated to whatever the final build is once we're live on nuget.org

**Pull request merged**